tests/onnx_backend_node_test.py

```diff
@@ -253,17 +253,65 @@ def run_node(cls,
 backend_test.exclude('test_operator_type_as_cpu')
 backend_test.exclude('test_tile_cpu')
 backend_test.exclude('test_tile_precomputed_cpu')
+backend_test.exclude('test_acos_cpu')
+backend_test.exclude('test_acos_example_cpu')
+backend_test.exclude('test_asin_cpu')
+backend_test.exclude('test_asin_example_cpu')
+backend_test.exclude('test_atan_cpu')
+backend_test.exclude('test_atan_example_cpu')
+backend_test.exclude('test_cos_cpu')
+backend_test.exclude('test_cos_example_cpu')
+backend_test.exclude('test_sin_cpu')
+backend_test.exclude('test_sin_example_cpu')
+backend_test.exclude('test_tan_cpu')
+backend_test.exclude('test_tan_example_cpu')
+backend_test.exclude('test_dropout_cpu')
+backend_test.exclude('test_dropout_default_cpu')
+backend_test.exclude('test_dropout_random_cpu')
+backend_test.exclude('test_gru_seq_length_cpu')
+backend_test.exclude('test_identity_cpu')
+backend_test.exclude('test_pow_bcast_scalar_cpu')
+backend_test.exclude('test_pow_bcast_array_cpu')
+backend_test.exclude('test_asin_example_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_default_axes_keepdims_example_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_default_axes_keepdims_random_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_do_not_keepdims_example_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_do_not_keepdims_random_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_keepdims_example_cpu')
+backend_test.exclude('test_reduce_log_sum_exp_keepdims_random_cpu')
+backend_test.exclude('test_rnn_seq_length_cpu')
+backend_test.exclude('test_operator_repeat_cpu')
+backend_test.exclude('test_operator_repeat_dim_overflow_cpu')
+backend_test.exclude('test_reduce_log_sum_default_cpu')
 
-# Supported ops, but Unsupported parameters by CoreML
-backend_test.exclude('test_thresholdedrelu_example_cpu')
 
-# maybe these can be fixed if we find a way to convert the "weight" input to a graph initializer field
-# otherwise they fall in the category of "supported ops, but unsupported parameters by CoreML"
+# Supported ops, but ONNX passes in dynamic inputs that CoreML needs to be constant graph initializers
+backend_test.exclude('test_thresholdedrelu_example_cpu')
+backend_test.exclude('test_batchnorm_epsilon_cpu')
+backend_test.exclude('test_batchnorm_example_cpu')
+backend_test.exclude('test_gemm_broadcast_cpu')
+backend_test.exclude('test_gemm_nobroadcast_cpu')
+backend_test.exclude('test_instancenorm_epsilon_cpu')
+backend_test.exclude('test_instancenorm_example_cpu')
+backend_test.exclude('test_prelu_example_cpu')
 backend_test.exclude('test_basic_conv_with_padding_cpu')
 backend_test.exclude('test_basic_conv_without_padding_cpu')
 backend_test.exclude('test_conv_with_strides_and_asymmetric_padding_cpu')
 backend_test.exclude('test_conv_with_strides_no_padding_cpu')
 backend_test.exclude('test_conv_with_strides_padding_cpu')
+backend_test.exclude('test_convtranspose_1d_cpu')
```

Comment:
just curious - hwy are they disabled? ConvTranspose is supported I guess - it's called Deconv

Reply:
The tests in this category are disabled because CoreML needs some input to be an attribute or a static initializer, but the onnx test cases define it to be a graph input.

Reply:
I think also all the 1D conv/pool test cases are disabled. They too need shape inference to work, for the converter to correctly map them to CoreML's conv (which is always 2D)